payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -413,6 +426,30 @@ data class PaymentMethodCreateParams internal constructor(
         }
     }
 
+    @Parcelize
+    data class USBankAccount(
+        internal var accountNumber: String,
+        internal var routingNumber: String,
+        internal var accountType: PaymentMethod.USBankAccount.USBankAccountType,
+        internal var accountHolderType: PaymentMethod.USBankAccount.USBankAccountHolderType
+    ) : StripeParamsModel, Parcelable {
```

Comment:
I assume that since this are the parameters to create the `PaymentMethod.USBankAccount` has extra fields that are only present in the return?  (fields like last4, bank name, etc)

Reply:
Yes that is correct